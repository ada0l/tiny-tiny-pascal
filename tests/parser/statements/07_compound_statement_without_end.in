begin
    begin
        begin
    end;
end.
