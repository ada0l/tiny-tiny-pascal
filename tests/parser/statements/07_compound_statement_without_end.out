Line: 5; Column: 4; Expected: SEMICOLON; but found .